conans/test/unittests/client/generators/premake_test.py

```diff
@@ -26,8 +26,8 @@ class PremakeGeneratorTest(unittest.TestCase):
     conan_bindirs = {{"{bin1}",
     "{bin2}"}}
     conan_libs = {{"libfoo", "libbar"}}
-    conan_cppdefines = {{"MYDEFINE2", "MYDEFINE1"}}
-    conan_cppflags = {{"-march=native", "-fPIE"}}
+    conan_defines = {{"MYDEFINE2", "MYDEFINE1"}}
+    conan_cxxflags = {{"-march=native", "-fPIE"}}
```

Comment:
```suggestion
    conan_cppflags = {{"-march=native", "-fPIE"}}
```

Reply:
Finally we agreed on having ``cpp_info.cppflags`` converted to conan_cxxflags. So no further action is required.